src/backend/joanie/settings.py

```diff
@@ -243,6 +279,10 @@ class Development(Base):
 class Test(Base):
     """Test environment settings"""
 
+    CACHES = {
+        "default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
+    }
+
```

Comment:
Could be remove as this is the cache used in Base?

Reply:
yes !